src/api/api.h

```diff
@@ -2,6 +2,7 @@
 #include <stdint.h>
 #include <stdarg.h>
 #include <stddef.h>
+#include "core/util.h"
```

Comment:
I'd like to avoid including util here if possible.

What are you using/exporting `lovrModules` for?  Could you use the new `luax_preload` function?  That function is meant to be an interface to have all the modules installed into a Lua state, without actually needing to know about the modules themselves.

Reply:
I'm replacing main.c with my own main function, so I'm using lovrModules to populate the Lua context. luax_preload sounds like exactly what I need :) I'll continue porting Alloverse to the dev branch, and potentially I can remove both the EXPORT stuff (it's only used for lovrModules right now anyway) AND the changes in this file. Watch this space :)